pandas/io/formats/format.py

```diff
@@ -1796,17 +1799,13 @@ def get_format_timedelta64(
 
     If box, then show the return in quotes
     """
-    values_int = values.view(np.int64)
+    values_int = np.asarray(values.view(np.int64))
```

Comment:
I think you shouldn't need to do this. once the overloads for view are more comprehensive, mypy will know that `.view(np.int64)` returns an numpy array.

I would leave this unchanged for now.